example/serializers.py

```diff
@@ -0,0 +1,37 @@
+"""
+REST API serializers for reading and writing to the edX Hedgehog Database.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from rest_framework import serializers
+
+
+# pylint: disable=abstract-method
+
+
+class HedgehogSerializer(serializers.Serializer):
+    """
+    Serializer for a hedgehog.
+    """
+    uuid = serializers.UUIDField()
+    key = serializers.SlugField()
+    name = serializers.CharField()
+    weight_grams = serializers.IntegerField()
+    # TODO: How can we make `weight_ounces` be schema'd as a float instead of a string?
+    weight_ounces = serializers.SerializerMethodField()
```

Comment:
Shouldn't we use a FloatField? https://www.django-rest-framework.org/api-guide/fields/#floatfield

Reply:
`weight_ounces` is a read-only field, calculated from `weight_grams`. The standard DRF way of implementing that is to make it a `SerializerMethodField`, which calls `get_{field_name}` to calculate the value. There doesn't seem to be a way in DRF to state that the method will return a `float`, so drf-yasg assumes that the field has type `str`.

However, I've realized that in Py3, we can use a type annotation, which drf-yasg is smart enough to pick up:
```python
class HedgehogSerializer(serializers.Serializer):
    ...
    weight_ounces = serializers.SerializerMethodField()
    ...
    def get_weight_ounces(self, obj) -> float:
        ...
```

So this becomes a Py2-only edge case, which I think is a non-issue.